### DISCUSSION TITLE
Gist Editor Does Not Utilize Screen Space

### DISCUSSION THREAD

**Discussion by 4piu** [community/community#78669] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

<img width="964" alt="image" src="https://github.com/community/community/assets/16668056/b7b43aba-0d8d-4586-a331-17d44e1ae0dd">

As the screenshot shows, the gist code editor has a fixed height and does not expand to utilize the screen space, making it difficult to work with.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @4piu! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, I will still submit your feedback to the relevant team here at GitHub so that this can be noted and filed. Please let me know if you have any additional questions! 

I've gone ahead and moved this to the correct category for you. Good luck!
